components/brave_wallet_ui/components/desktop/popup-modals/add-account-modal/add-hardware-account-modal.tsx

```diff
@@ -39,7 +39,11 @@ interface Params {
   accountTypeName: string
 }
 
-export const AddHardwareAccountModal = () => {
+interface Props {
+  isOnboarding?: boolean | undefined
```

Comment:
How about a `onSelectAccountType` prop instead to prevent writing conditional logic within this component as to how selections should be handled?